playbooks/roles/foreman_proxy_content/tasks/koji_repos.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Would an install not be able to use the `katello_repositories` role to achieve this?